<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1104</width>
    <height>786</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-image: url(:/ttdyelloclore.jpg);</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>446</x>
      <y>0</y>
      <width>91</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(:/new/prefix1/TTD_IMG/TTD-Logo.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>20</y>
      <width>111</width>
      <height>121</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(:/leftside.jpg);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>160</y>
      <width>911</width>
      <height>551</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="page">
     <widget class="QLineEdit" name="Name">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>59</y>
        <width>201</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>MS Shell Dlg 2</family>
        <pointsize>11</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-image: url(:/whitte.jpg);
color: rgb(0, 0, 0);
font: 11pt &quot;MS Shell Dlg 2&quot;;</string>
      </property>
      <property name="maxLength">
       <number>30</number>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>70</y>
        <width>47</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>Name</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="Mobile">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>110</y>
        <width>201</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>MS Shell Dlg 2</family>
        <pointsize>11</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-image: url(:/whitte.jpg);
color: rgb(0, 0, 0);
font: 11pt &quot;MS Shell Dlg 2&quot;;</string>
      </property>
      <property name="maxLength">
       <number>10</number>
      </property>
      <property name="frame">
       <bool>true</bool>
      </property>
      <property name="cursorPosition">
       <number>0</number>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label1">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>120</y>
        <width>61</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>Mobile </string>
      </property>
     </widget>
     <widget class="QComboBox" name="state">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>160</y>
        <width>201</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="iconSize">
       <size>
        <width>40</width>
        <height>40</height>
       </size>
      </property>
      <item>
       <property name="text">
        <string>state</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Andhra Pradesh (AP)</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Arunachal Pradesh (AR)</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Assam (AS)</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Bihar (BR)</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Chhattisgarh (CG)</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Goa (GA)</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Gujarat (GJ)</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Haryana (HR)</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Himachal Pradesh (HP)</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Jammu and Kashmir (JK)</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Jharkhand (JH)</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Karnataka (KA)</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Kerala (KL)</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Madhya Pradesh (MP)</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Maharashtra (MH)</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Manipur (MN)</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Meghalaya (ML)</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Mizoram (MZ)</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Nagaland (NL)</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Odisha(OR)</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Punjab (PB)</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Rajasthan (RJ)</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Sikkim (SK)</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Tamil Nadu (TN)</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Telangana (TS)</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Tripura (TR)</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Uttar Pradesh (UP)</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Uttarakhand (UK)</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>West Bengal (WB)</string>
       </property>
      </item>
     </widget>
     <widget class="QLabel" name="label3">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>170</y>
        <width>81</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string> State</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="Aadhaar">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>210</y>
        <width>201</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>MS Shell Dlg 2</family>
        <pointsize>11</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-image: url(:/whitte.jpg);
color: rgb(0, 0, 0);
font: 11pt &quot;MS Shell Dlg 2&quot;;</string>
      </property>
      <property name="maxLength">
       <number>16</number>
      </property>
     </widget>
     <widget class="QLabel" name="label4">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>210</y>
        <width>131</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>Aadhaar Nunber</string>
      </property>
     </widget>
     <widget class="QLabel" name="label5">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>280</y>
        <width>141</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>MS Shell Dlg 2</family>
        <pointsize>12</pointsize>
        <weight>9</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);
font: 75 12pt &quot;MS Shell Dlg 2&quot;;</string>
      </property>
      <property name="text">
       <string>No. of Items/Bags</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="item">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>270</y>
        <width>201</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>MS Shell Dlg 2</family>
        <pointsize>11</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-image: url(:/whitte.jpg);
color: rgb(0, 0, 0);
font: 11pt &quot;MS Shell Dlg 2&quot;;</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="maxLength">
       <number>3</number>
      </property>
     </widget>
     <widget class="QPushButton" name="Submit">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>440</y>
        <width>131</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-image: url(:/buttonclore.jpg);
color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string> Print</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_2">
      <property name="geometry">
       <rect>
        <x>510</x>
        <y>440</y>
        <width>111</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 12pt &quot;MS Shell Dlg 2&quot;;
color: rgb(255, 255, 255);
background-image: url(:/buttonclore.jpg);
</string>
      </property>
      <property name="text">
       <string>TablesView</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>330</y>
        <width>201</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>MS Shell Dlg 2</family>
        <pointsize>11</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-image: url(:/whitte.jpg);
color: rgb(0, 0, 0);
font: 11pt &quot;MS Shell Dlg 2&quot;;</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>340</y>
        <width>121</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 12pt &quot;MS Shell Dlg 2&quot;;
color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>Token Number</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>20</y>
        <width>241</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 75 12pt &quot;MS Shell Dlg 2&quot;;
color: rgb(170, 0, 0);</string>
      </property>
      <property name="text">
       <string>Enter Pilgrim luggage Details:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>660</x>
        <y>80</y>
        <width>181</width>
        <height>101</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">image: url(:/jvt.jpg);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>660</x>
        <y>190</y>
        <width>211</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(0, 0, 255);
font: 10pt &quot;MS Shell Dlg 2&quot;;</string>
      </property>
      <property name="text">
       <string>Jai Venkateswara Technologies</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_3">
     <widget class="QTextEdit" name="textEdit">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>40</y>
        <width>711</width>
        <height>441</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 12pt &quot;MS Shell Dlg 2&quot;;</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_3">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>410</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>print</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_4">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>410</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Edit</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_4">
     <widget class="QTableWidget" name="tableWidget">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>90</y>
        <width>721</width>
        <height>321</height>
       </rect>
      </property>
      <column>
       <property name="text">
        <string>id</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Name</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Mobileno</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>state</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Aadherno</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>itemno</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>tockenid</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>date</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>time</string>
       </property>
      </column>
     </widget>
     <widget class="QPushButton" name="pushButton_5">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>450</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 75 12pt &quot;MS Shell Dlg 2&quot;;
color: rgb(255, 0, 0);</string>
      </property>
      <property name="text">
       <string>Back</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="search_line">
      <property name="geometry">
       <rect>
        <x>560</x>
        <y>20</y>
        <width>141</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_6">
      <property name="geometry">
       <rect>
        <x>730</x>
        <y>20</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 75 11pt &quot;MS Shell Dlg 2&quot;;
color: rgb(255, 0, 0);</string>
      </property>
      <property name="text">
       <string>Search</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>600</x>
        <y>450</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 75 11pt &quot;MS Shell Dlg 2&quot;;
color: rgb(255, 0, 0);</string>
      </property>
      <property name="text">
       <string>Refresh</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>40</y>
      <width>121</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(:/rightside.jpg);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>20</y>
      <width>451</width>
      <height>111</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(:/center.jpg);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="signout">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>132</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(170, 0, 0);
font: 75 italic 8pt &quot;MS Sans Serif&quot;;</string>
    </property>
    <property name="text">
     <string>Signout</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1104</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
